app/api/v2/handlers/operation_api.py

```diff
@@ -47,3 +74,24 @@ async def get_operation_report(self, request: web.Request):
         access = await self.get_request_permissions(request)
         report = await self._api_manager.get_operation_report(operation_id, access)
         return web.json_response(report)
+
+    '''Overridden Methods'''
+    async def create_object(self, request: web.Request):
+        data = await request.json()
+        await self._error_if_object_with_id_exists(data.get(self.id_property))
+        access = await self.get_request_permissions(request)
+        return await self._api_manager.create_object_from_schema(self.schema, data, access)
+
+    async def create_or_update_object(self, request: web.Request):
+        data, access, obj_id, query, search = await self._parse_common_data_from_request(request)
+        matched_obj = self._api_manager.find_object(self.ram_key, query)
+        if matched_obj and matched_obj.access not in access['access']:
+            raise JsonHttpForbidden(f'Cannot update {self.description} due to insufficient permissions: {obj_id}')
+        return await self._api_manager.create_object_from_schema(self.schema, data, access, matched_obj)
```

Comment:
Shouldn't `create_object_from_schema` be called only if making a new operation? It looks like we're using it to try to handle operation updates, but then new operations would get created each time.

Reply:
I'm wondering if we can use the following logic:
```
if operation exists based on ID:
   if we have access: call self.update_operation(request). This should allow us to take advantage of that decorator and its checks. We may need edit update_operation to do the state checks 
   if we don't have access: return forbidden message
else:
   create new operation as before
```

Reply:
Right now this is following the same logic as the original base_object_api.py method, essentially PUT requests just function as a create/replace method instead of create/update

Reply:
After looking through everything a bit, it seems that the once an operation bucket/link is picked up, the server seems to block until that is complete. After that, the operation can be replaced. 

That being said, the old api used PUT as another POST essentially (at least in terms of the functions called). It didn't allow for replacement of an operation, it would simply create a new one. PATCH only allows for state/autonomous/obfuscator fields to be updated. So with that in mind, does it even make sense to keep the PUT request?

Reply:
Resolved: We decided to remove the PUT request as it's unnecessary for this endpoint given its usage in the UI